tests/php/bootstrap.php

```diff
@@ -35,4 +35,20 @@
 error_reporting(E_ALL & ~E_STRICT);
 
 require_once realpath(__DIR__ . '/polyfill.php');
-require_once realpath(__DIR__ . '/../../vendor/autoload.php');
+$loader = require realpath(__DIR__ . '/../../vendor/autoload.php');
+
+Doctrine\Common\Annotations\AnnotationRegistry::registerLoader([$loader, 'loadClass']);
+
+global $dependencyInjector;
```

Comment:
why declare global variable here?

Reply:
the same as here https://github.com/centreon/centreon/blob/master/bootstrap.php#L58 :)